integration_test/third_party_apps_data/applications/dcgm/centos_rhel/install

```diff
@@ -1,7 +1,23 @@
 set -e
 source /etc/os-release
 KERNEL_VERSION=`uname -r`
-sudo yum install -y kernel-devel-${KERNEL_VERSION} pciutils gcc make wget yum-utils 
+
+# sudo yum install -y kernel-devel-${KERNEL_VERSION} pciutils gcc make wget yum-utils 
+# TODO(b/312455563): Temporary fix for Rocky Linux 9.2/8.8; uncomment the above line and remove this following section once the image has been updated to Rocky Linux 9.3/8.9
+sudo yum install -y pciutils gcc make wget yum-utils
+if ! sudo yum install -y kernel-devel-${KERNEL_VERSION}; then 
+    sudo yum install -y kernel-devel  # Install the latest kernel dev package first to bring in the dependencies, before switching to 9.2 vault repo
+    if [[ $ID == rocky && "${VERSION_ID}" == 9.2 ]]; then
+        sudo sed -i 's,mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=$basearch&repo=AppStream-$releasever$rltype,mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=x86_64\&repo=AppStream-9.2,g' /etc/yum.repos.d/rocky.repo
```

Comment:
I'm confused, why are we not using `$basearch` and instead using `arch=x86_64` here?

Reply:
Done! Good point - we only need to change the release version in the url.